app-server/app/src/main/resources/application.conf

```diff
@@ -53,4 +53,7 @@ auth0 {
   # Development Secret Only
   secret = ""
   secret = ${?AUTH0_CLIENT_SECRET}
+  # Auth0 Bearer with proper permissions for Management API
+  bearer = ""
+  bearer = ${?AUTH0_MANAGEMENT_BEARER}
```

Comment:
If we need permissions for other things does it make sense to update this token or create a different one?